ui/app/pages/settings/networks-tab/network-form/network-form.component.js

```diff
@@ -0,0 +1,225 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import validUrl from 'valid-url'
+import PageContainerFooter from '../../../../components/ui/page-container/page-container-footer'
+import TextField from '../../../../components/ui/text-field'
+
+export default class NetworksTab extends PureComponent {
+  static contextTypes = {
+    t: PropTypes.func,
```

Comment:
```suggestion
    t: PropTypes.func.isRequired,
```

Reply:
I think we should try to add `.isRequired` to all appropriate prop types going forward. I've suggested them here for the `contextTypes` only since I _know_ that these are required and suggesting it for the correct props would require more thought—if you want to add them quickly go for it, but this is more a "moving forward" suggestion.

Reply:
fixed in 29c5762e6